### ISSUE TITLE
What is actually needed to ensure no visual glitches at representation switches due to resolution/aspectratio?

### ISSUE THREAD

**Issue by sandersaares** [Dash-Industry-Forum/DASH-IF-IOP#284]
There is text in the first paragraph of 6.2.5.1 about ensuring that switching between video representations does not cause glitches. I understand the goal but I do not understand the description.

> Video Adaptation Sets may contain Representations encoded at lower resolutions that are exactly divisible subsamples of the source image size.  As a result, the cropped vertical and horizontal sample counts of all Representations can be scaled to a common display size without position shift or aspect ratio distortion that would be visible during adaptive switching.  Subsample ratios must result in integer values for the resulting encoded sample counts (without rounding or truncation). The encoded sample count shall scale to the source video’s exact active image aspect ratio when combined with the encoded sample aspect ratio value aspect_ratio_idc stored in the video Sequence Parameter Set NAL

I do not understand what the following sentence fragments mean:

* exactly divisible subsamples
* subsample ratios must result in integer values for the resulting encoded sample counts

I understand the last sentence there - make sure that multiplying the pixels with the sample aspect ratio results in the proper display aspect ratio - but the rest is a mystery to me.

Furthermore, I have never paid any attention to this "exactly divisible" thing (whatever it means) and have always used ballpark figures for my resolutions and yet none of my videos have ever experienced glitches at representation switches - is this really relevant? As far as my practical experience goes, as long as the sample aspect ratio contains the right value, it does not matter how many pixels you encode.

**Comment by RufaelDev** [comment]
I think referring to CMAF may help, as CMAF switching sets have some features that can help as it takes into account these visual disturbances aspect ratio, referencing may help avoid text that is not clear in the IOP. The divisible is needed to maintain exactly the same aspect ratio between different bit-rate tracks which may cause visual disturbance. Again, referring to text may help the IOP as it seems few people are working on the text.

**Comment by haudiobe** [comment]
I agree that we should use references to CMAF and CTA WAVE for this.
